app/src/main/java/org/sil/gatherwords/MainActivity.java

```diff
@@ -96,6 +98,8 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 Intent i = new Intent(this, PreferencesActivity.class);
                 startActivity(i);
                 return true;
+            case R.id.undo_session_delete:
+                new UndoDeleteSessionFromDB(this).execute();
             default:
```

Comment:
Is fall through behavior your intent? If so, please add a comment saying so.

Reply:
What is fall through behavior?

Reply:
After line 102 executes, code flow will continue, also executing everything in the `default:` block (not exiting the switch).

Reply:
Don't foresee a problem with fall through, but added a return true just in case.